modules/nf-core/lofreq/alnqual/main.nf

```diff
@@ -0,0 +1,51 @@
+process LOFREQ_ALNQUAL {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/lofreq:2.1.5--py310h47ef89e_10' :
+        'biocontainers/lofreq:2.1.5--py310h47ef89e_10' }"
+
+    input:
+    tuple val(meta), path(bam)
```

Comment:
Does this also support SAM and CRAM files? otherwise it would maybe be better to use something more general as input name (like `alignment` or `input`...)

Reply:
Doesn't support cram files. May technically support sam files, but nobody should be using those anyway...